### DISCUSSION TITLE
How can I use ErrorMessage with validate?

### DISCUSSION THREAD

**Discussion by REDLIGHTFILMS** [react-hook-form/react-hook-form#4547] [Q&A]
How can I use ErrorMessage with validate? I want to pass into the validate function the value, from the text field and then validate. How can I do that?

validate: (value: string) => { if(Number(value) > 3) { return "test" } }

This code does not work....

**Comment by jorisre** [comment]
Hi @REDLIGHTFILMS 

Can you share a codesandbox with your problem please ?
